contrib/scripts/check-k8s-code-gen.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+# Delete all zz_generated.deepcopy.go files
+find . -not -path "./vendor/*" -iname *"zz_generated.deepcopy.go" -exec rm  {} \;
```

Comment:
```suggestion
find . -not -path "./vendor/*" -iname "*zz_generated.deepcopy.go" -exec rm  {} \;
```

Did you mean this instead?

Reply:
yes :facepalm: